app/services/achiever/course/template.rb

```diff
@@ -22,7 +22,8 @@ class Achiever::Course::Template
     :topics_covered,
     :who_is_it_for,
     :workstream,
-    :always_on
+    :always_on,
+    :is_new_course
```

Comment:
Would it be nicer to have a method called `new_course?` that returns the value, rather than `is_new_course`?